test/srt/test_vision_openai_server.py

```diff
@@ -699,6 +699,8 @@ def setUpClass(cls):
                 "--trust-remote-code",
                 "--chat-template",
                 "gemma-it",
+                "--mem-fraction-static",
+                "0.7",
```

Comment:
We should not change ut, let the default value be OK, such as a default value of 0.75?

Reply:
@zhyncs Do you want me to set the `--mem-fraction-static` to be `0.75` here? Although I don't know what is the default value of `--mem-fraction-static` for this case, but it may be `0.88` if it's on a single GPU?
https://github.com/sgl-project/sglang/blob/42a45df043f5f381bc3f3c93afef10d4b98153c6/python/sglang/srt/server_args.py#L227-L228

Reply:
I just updated the value to `0.75`

Reply:
or we can limit the `--context-length` to be something like `4096` and might work as well?

Reply:
@zhyncs let me know what you think!

Reply:
I think we should not limit the length of the context.